mountpoint-s3-client/examples/client_benchmark.rs

```
@@ -198,7 +199,7 @@ struct CliArgs {
     #[arg(
         long,
         help = "Desired throughput in Gbps",
-        default_value_t = 10.0,
+        default_value_t = 100.0,
```

Unrelated to the PR? Maybe we should just error out if users do not provide throughput for these benchmarks (at least when not mocked) -- silently defaulting / changing the default seems like something that could cause results to vary without being obvious.